FFXIV_TexTools_CLI/Arguments.cs

```diff
@@ -188,5 +189,53 @@ public void ArgumentHandler(string[] args)
                     break;
             }
         }
+
+        bool BackupStatus()
+        {
+            bool keepGoing = false;
+            if (MainClass._backupDirectory == null)
+                main.PrintMessage("No backup directory specified, can't check the status of backups", 2);
```

Comment:
+
"You are strongly recommended to add a backup directory in $configfilepath"?

Reply:
e0caa10 and 7573cc1 now add these lines to the warning when no backup directory is detected. It will now also inform you that it is checking your backup files before proceeding with your requested course of action.